backend/src/api/auth/auth.controller.js

```diff
@@ -59,3 +62,53 @@ exports.loginUser = async (req, res, next) => {
     token: token,
   });
 };
+
+exports.authorize = async (req, res, next) => {
```

Comment:
But... why? We already have authorization middleware in utils folder. Please be careful next time and don't do unnesesary work

Reply:
ok